static/styles/modal.css

```diff
@@ -86,7 +86,6 @@
     border-radius: 0.25rem;
     border-width: 0;
     cursor: pointer;
-    appearance: button;
```

Comment:
@timabbott I would be interested in merging this change since this changes appearance of buttons based on OS in [theory](https://developer.mozilla.org/en-US/docs/Web/CSS/appearance). 

> The appearance CSS property is used to control native appearance of UI controls, that are based on operating system's theme.